docs/generated/packages/eslint-plugin/documents/dependency-checks.md

```diff
@@ -8,7 +8,11 @@ We use the version numbers of the installed packages when checking whether the v
 
 ## Usage
 
-You can use the `dependency-checks` rule by adding it to your ESLint rules configuration:
+Library generators from `@nx` packages will configure this rule automatically when you opt-in for bundler/build setup.
+
+### Manual setup
+
+To set it up manually for existing libraries, you need to add the `dependency-checks` rule to your project's ESLint configuration:
 
 ```jsonc {% fileName=".eslintrc.json" %}
```

Comment:
should we add here 

`{% fileName="<your-project-root>/.eslintrc.json" %}`

Just to make it more clear this should be in your project rather than the root one?